test/cppsuite/test_harness/workload/database_operation.cpp

```diff
@@ -380,9 +380,14 @@ database_operation::update_operation(thread_context *tc)
 }
 
 void
-database_operation::validate(const std::string &operation_table_name,
+database_operation::validate(bool custom_tracking_table, const std::string &operation_table_name,
   const std::string &schema_table_name, const std::vector<uint64_t> &known_collection_ids)
 {
+    if (custom_tracking_table)
+        testutil_die(EINVAL,
+          "Attempting to perform default validation on a test with a user-defined tracking "
```

Comment:
We don't need to say "Please define custom validation". Simply say "Please define validation".

Reply:
Fixed in 325247ff2f0d2cdd6175e87c536d66b735e51806